estimation/estimate_cybershake.py

```diff
@@ -155,8 +155,9 @@ def run_estimations(
 
     if hf_input_data is not None:
         print("Running HF estimation")
-        hf_core_hours, hf_run_time = estimate_wct.estimate_HF_chours(hf_input_data)
-        hf_cores = hf_input_data[:, -1]
+        hf_core_hours, hf_run_time, hf_cores = estimate_wct.estimate_HF_chours(
+            hf_input_data, True
```

Comment:
i would suggest using
```
scale_ncores = True
```
otherwise I have no idea what True represents before i dig into the code and find the function